pages/linux/wodim.md

```diff
@@ -1,14 +1,13 @@
 # wodim
 
-> Command for record/burning data using a optical drive to CD or DVD.
-> This command is also called cdrecord on some systems, but it is wodim underneath.
-> Some usage of wodim can cause destructive actions, such as erasing all the data on a disc.
+> Command (aliased as `cdrecord` on some systems) for recording data to CDs or DVDs.
+> Some invocations of wodim can cause destructive actions, such as erasing all the data on a disc.
 
-- Display optical drives available to wodim:
+- Display optical drives available to `wodim`:
 
 `wodim --devices`
 
-- Record pure CD-DA audio to a disc:
+- Record ("burn") pure CD-DA audio to a disc:
```

Comment:
`pure CD-DA audio` -> `a pure CD-DA audio file`?

Reply:
Yeah, that makes it clearer, thanks. I'm not sure what "pure" means in this context, though. Any hints, @Nagiza, @agnivade?

Reply:
As i understand it "pure" means that the burned CD only contains audio tracks and no data track, such as a album cover.
I found that the "CD-DA" is a unnecessary detail. 
`Record ("burn") pure CD-DA audio to a disc:` -> `Record ("burn") a audio only disc:`

Reply:
Nice, that's even more readable. I'll change when I'm back on a desktop.